File: 1-two-sum/two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        dic = []
        it = 0
        for i in range(len(nums)):
            for x in range(i+1 , len(nums)):
                if nums[i] + nums[x] == target:
                    dic.append(i)
                    dic.append(x)
                    return dic
        

       
        

File: 1-two-sum/two-sum.php
class Solution {

    /**
     * @param Integer[] $nums
     * @param Integer $target
     * @return Integer[]
     */
    function twoSum($nums, $target) {
        
        for ($i = 0; $i < sizeof($nums); $i++) {

            for ($j=1; $j <sizeof($nums); $j++){

                if( ($nums[$i] + $nums[$j] == $target) && $i !== $j){
                    return [$i , $j];
                }
            }
            
        }
    }
}